python/lsst/ts/ATWhiteLightSource/chillerComponent.py

```diff
@@ -5,30 +5,40 @@
 
 
 class ChillerComponent(object):
-    def __init__(self, ip, port):
+    def __init__(self, ip, port, com_lock, log=None):
         self.ip = ip
         self.port = port
         self.connect_task = None
+        self.log = None
         self.reader = None
         self.writer = None
-        self.timeout = 5
+        self.timeout = 15
         self.response_dict = {}
         self.last_response = None
-        self.chiller_com_lock = asyncio.Lock()
+        self.chiller_com_lock = com_lock
 
     async def connect(self):
-        """Connect to chiller's ethernet-to-serial bridge"""
-        #self.log.debug(f"connecting to: {self.ip}:{self.port}.")
+        """Connect to chiller's ethernet-to-serial bridge"""
+        # self.log.debug(f"connecting to: {self.ip}:{self.port}.")
         if self.connected:
-            raise RuntimeError("Already connected")
-        print("about to connect to "+str(self.ip))
-        self.reader, self.writer = await asyncio.wait_for(asyncio.open_connection(self.ip, self.port), self.timeout)
-        print("done connecting")
+            raise RuntimeError("Already connected")
+
+        self.log.debug("about to connect to chiller at"+str(self.ip))
+        try:
+            self.reader, self.writer = await asyncio.wait_for(asyncio.open_connection(self.ip, self.port), self.timeout)
+        except Exception as e:
```

Comment:
Not sure I understood the need to capture this exception. You are logging it and then raising the exception so I don't see the point.